frontend/rolecall/src/app/cast/cast-editor-v2.component.ts

```diff
@@ -94,10 +102,26 @@ export class CastEditorV2 implements OnInit {
     }
   }
 
+  buildPopupList() {
+    this.popupList = this.allPieces.map((piece, pieceIx) => {
+      const existingCasts = this.allCasts.filter((val) => val.segment == piece.uuid);
+      const prefix = existingCasts.length === 0 ? " " : "z";
+      const name = existingCasts.length === 0 ? "*" + piece.name : piece.name;
```

Comment:
What does the "*" mean? If it's going to be displayed in the menu as part of the actual name then it's an accessibility issue (though not sure how much that's important to us right now)

Reply:
See the comment above. The problem was that the customer wanted an indication of which ballets had not been casted (or finished). Sorting these at the to and preceding the ballet name with a '*' seemed like a great improvement over what was before, and probably one that doesn't need to be changed going forward. (If we get accessibility issues, perhaps we can test for the leading *.)

Reply:
Ack, I guess let's leave it as is for now and take another look later.